MAVProxy/modules/mavproxy_magical/data/arrow.mtl

```diff
@@ -0,0 +1,14 @@
+# Magical Arrow Material (c) by Intel Corporation
+#
+# Magical Arrow Material is licensed under a
+# Creative Commons Attribution-ShareAlike 4.0 International License.
+#
+# You should have received a copy of the license along with this
```

Comment:
Are there Open or Free tools available that can manipulate these wavefront data objects easily and well?  I can see several that claim to...